### DISCUSSION TITLE
batch response: Post https://lfs.github.com/.../objects/batch: malformed HTTP status code "Ubuntu-4ubuntu0.11"

### DISCUSSION THREAD

**Discussion by zchwang** [git-lfs/git-lfs#5626] [Q&A]
Encountered the following error while uploading a large file (git push).

batch response: Post https://lfs.github.com/.../objects/batch: malformed HTTP status code "Ubuntu-4ubuntu0.11"

What is the cause of this?

**Comment by bk2204** [comment]
It sounds like you're trying to talk HTTPS to something that isn't an HTTPS endpoint (maybe it's an SSH endpoint).  Are you using a proxy of some sort (including a non-default antivirus or firewall)?

You can see more output from running your push with `GIT_TRACE=1 GIT_TRANSFER_TRACE=1 GIT_CURL_VERBOSE=1`.
